pom.xml

```diff
@@ -390,6 +391,12 @@
             </dependency>
 
             <dependency>
+                <groupId>commons-lang</groupId>
+                <artifactId>commons-lang</artifactId>

```

Comment:
Is comomns-lang still needed now that the escape code is MessageSanitizerContainerResponseFilter ?